README.md

```diff
@@ -107,9 +107,9 @@ vars:
 
 ### Passing Through Additional Fields
 
-This package includes fields we judged were standard across Iterable users. However, the Fivetran connector allows for additional columns to be brought through in the `event_extension` and `user_history` objects. Therefore, if you wish to bring them through, leverage our passthrough column variables. 
+This package includes fields we judged were standard across Iterable users. However, the Fivetran connector allows for additional columns to be brought through in the `event_extension` and `user_history` objects. Therefore, if you wish to bring them through, leverage our passthrough column variables. For event_extension columns, ensure that `iterable__using_event_extension` is set to True, which is the default.
 
-You will see these additional columns populate in the end `iterable__events` and `iterable__users` models.
+You will see these additional columns populate in the end `iterable__list_user_history`, `iterable__events`, and `iterable__users` models.
```

Comment:
Is this meant to be `iterable__user_campaign`?

Reply:
`iterable__user_campaign` doesn't involve any pass through columns I believe.